packages/guideline-blocks-settings/src/components/RichTextEditor/plugins/TextStylePlugins/quotePlugin.tsx

```diff
@@ -42,7 +42,7 @@ export const QuoteMarkupElementNode = ({ element, attributes, children, styles }
     return (
         <blockquote
             {...attributes}
-            className={merge([align && alignmentClassnames[align], getColumnBreakClasses(element), 'quote'])}
+            className={merge([align && alignmentClassnames[align], getColumnBreakClasses(element), 'a-quote'])}
```

Comment:
why `a-quote`?

Reply:
What is `a-quote` representing? from where is that class coming from?

Reply:
We use it to to inherit the right font-family if "quote" has the font-family `inherit`.

It comes from app-server's `appearance.php`

Reply:
This is really flakey as we are not aware of it being used here, (we are currently on the path of removing the file), please rely on the CSS variables only

Reply:
(To be tackled in follow up PR)